src/components/Personalization/index.js

```diff
@@ -168,14 +167,10 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
     commands: {
       getDecisions(options = {}) {
```

Comment:
I think you can remove the cloning, if the `scopes` parameter is not modified.

Reply:
Indeed, the validation effectively clones the array.